DataFormats/Detectors/TRD/include/DataFormatsTRD/T0FitHistos.h

```diff
@@ -0,0 +1,61 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file T0FitHistos.h
+/// \brief Class to store the TRD PH values for each TRD chamber
+
+#ifndef ALICEO2_T0FITHISTOS_H
+#define ALICEO2_T0FITHISTOS_H
+
+#include "DataFormatsTRD/Constants.h"
+#include "DataFormatsTRD/PHData.h"
+#include "Framework/InputRecord.h"
+#include "Rtypes.h"
+#include <vector>
+#include <memory>
+#include <gsl/span>
+
+namespace o2
+{
+namespace trd
+{
+
+class T0FitHistos
+{
+ public:
+  T0FitHistos() = default;
+  T0FitHistos(const T0FitHistos&) = default;
+  ~T0FitHistos() = default;
+  void init();
```

Comment:
I don't think this `init()` method is needed

Reply:
removed init()